docs/build_parameters.rst

```diff
@@ -254,10 +252,37 @@ to the root of the object::
                         "required": ["openidc_dir"]
                     }
                 ]
+            },
+            "signing_intents": {
+                "description": "List of signing intents",
+                "type": "array",
+                "items": {
+                    "properties": {
+                      "name": {
+                        "description": "signing intent name",
```

Comment:
Could this be made uniform, and start descriptions with a capital letter? (And not ending them with a period.)